Cannot build fresh copy of Manifold.

Here is how I am building it:
```sh
cd manifold
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Release -DMANIFOLD_JSBIND=OFF -DMANIFOLD_CBIND=ON -DMANIFOLD_PYBIND=ON -DMANIFOLD_EXPORT=OFF -DBUILD_SHARED_LIBS=OFF -DMANIFOLD_TEST=ON .. && make
test/manifold_test
```
Here is the output:
```
-- The CXX compiler identification is GNU 11.4.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.2") 
-- Checking for module 'Clipper2'
--   No package 'Clipper2' found
-- clipper2 not found, downloading from source
-- The C compiler identification is GNU 11.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
[ 11%] Creating directories for 'glm-populate'
[ 22%] Performing download step (git clone) for 'glm-populate'
Cloning into 'glm-src'...
remote: Enumerating objects: 17506, done.        
remote: Counting objects: 100% (17506/17506), done.        
remote: Compressing objects: 100% (5533/5533), done.        
remote: Total 17506 (delta 15332), reused 13464 (delta 11921), pack-reused 0        
Receiving objects: 100% (17506/17506), 34.44 MiB | 33.49 MiB/s, done.
Resolving deltas: 100% (15332/15332), done.
fatal: reference is not a tree: b06b775c1c80af51a1183c0e167f9de3b2351a79
CMake Error at glm-subbuild/glm-populate-prefix/tmp/glm-populate-gitclone.cmake:40 (message):
  Failed to checkout tag: 'b06b775c1c80af51a1183c0e167f9de3b2351a79'


gmake[2]: *** [CMakeFiles/glm-populate.dir/build.make:102: glm-populate-prefix/src/glm-populate-stamp/glm-populate-download] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/glm-populate.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2

CMake Error at /usr/share/cmake-3.22/Modules/FetchContent.cmake:1087 (message):
  Build step for glm failed: 2
Call Stack (most recent call first):
  /usr/share/cmake-3.22/Modules/FetchContent.cmake:1216:EVAL:2 (__FetchContent_directPopulate)
  /usr/share/cmake-3.22/Modules/FetchContent.cmake:1216 (cmake_language)
  /usr/share/cmake-3.22/Modules/FetchContent.cmake:1259 (FetchContent_Populate)
  manifoldDeps.cmake:45 (FetchContent_MakeAvailable)
  CMakeLists.txt:68 (include)


-- Configuring incomplete, errors occurred!
See also "/home/brian/work/manifold/manifold/build/CMakeFiles/CMakeOutput.log".
./build_manifold: line 5: test/manifold_test: No such file or directory
```

I'm experiencing the same problem on Windows; will update if I figure anything out 🙂👍

```
  fatal: reference is not a tree: b06b775c1c80af51a1183c0e167f9de3b2351a79
  CMake Error at glm-subbuild/glm-populate-prefix/tmp/glm-populate-gitclone.cmake:49 (message):
    Failed to checkout tag: 'b06b775c1c80af51a1183c0e167f9de3b2351a79'
```

--- a/manifoldDeps.cmake
+++ b/manifoldDeps.cmake
@@ -21,7 +21,6 @@ else()
     FetchContent_Declare(Clipper2
         GIT_REPOSITORY https://github.com/AngusJohnson/Clipper2.git
         GIT_TAG Clipper2_1.3.0
-        GIT_SHALLOW TRUE
         GIT_PROGRESS TRUE
         SOURCE_SUBDIR CPP
     )
@@ -38,7 +37,6 @@ endif()
 FetchContent_Declare(glm
     GIT_REPOSITORY https://github.com/g-truc/glm.git
     GIT_TAG b06b775c1c80af51a1183c0e167f9de3b2351a79
-    GIT_SHALLOW TRUE
     GIT_PROGRESS TRUE
     FIND_PACKAGE_ARGS NAMES glm
 )
@@ -61,7 +59,6 @@ if(MANIFOLD_PAR STREQUAL "TBB" AND NOT TBB_FOUND)
     FetchContent_Declare(TBB
         GIT_REPOSITORY https://github.com/oneapi-src/oneTBB.git
         GIT_TAG        v2021.11.0
-        GIT_SHALLOW    TRUE
         GIT_PROGRESS   TRUE
     )
     FetchContent_MakeAvailable(TBB)

